src/shared/util.js

```diff
@@ -634,19 +634,22 @@ class FeatureTest {
       (typeof navigator !== "undefined" &&
         typeof navigator?.platform === "string")
     ) {
+      const isFirefox =
+        (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) ||
+        (typeof navigator?.userAgent === "string" &&
+          navigator.userAgent.includes("Firefox"));
       return shadow(this, "platform", {
+        isChrome: !isFirefox && !!window.chrome,
```

Comment:
Unfortunately this isn't safe, since code in `src/shared/` will run on both the main- and worker-threads (and in Node.js) so you cannot access `window` here.

Can we please just revert the changes in this file (I'll make a few other suggestions)?